addons/stock/models/stock_location.py

```diff
@@ -356,8 +356,8 @@ def _check_package_storage(self, product, package):
         return True
 
 
-class Route(models.Model):
-    _name = 'stock.location.route'
+class StockRoute(models.Model):
+    _name = 'stock.route'
```

Comment:
Could be a good opportunity to move the `stock.route` code in a specific file? :)